src/rdkafka.h

```diff
@@ -7085,6 +7127,46 @@ typedef enum rd_kafka_AclOperation_t {
 /**@}*/
 
 
+/**
+ * @name ListOffsets API
+ * @brief Given a topic_partition list, provides the offset information.
+ * @{
+*/
+
+/**
+ * OffsetSpec for Earliest, Latest and Timestamp of the Offset to list via ListOffsets.
+*/
+typedef enum rd_kafka_OffsetSpec_t{
+    RD_KAFKA_OFFSET_SPEC_MAX_TIMESTAMP = -3,
```

Comment:
Document the values, for example about this one (javadoc):
```
Used to retrieve the offset with the largest timestamp of a partition as message timestamps can be specified client side this may not match the log end offset returned by SPEC_LATEST.
```